MovingBlocks/Terasology
# fix: enable block tinting in LOD chunks

### Contains

Block tinting previously queried biome information from the `WorldProvider`. But LOD chunks are never added to the world, just meshed and rendered. All the necessary data is still there, so this switches to passing block tinting systems a `ChunkView` from which to derive necessary information, which really makes more sense anyway.

### How to test

With https://github.com/Terasology/BiomesAPI/pull/18, try enabling LOD chunks in Core Gameplay or Metal Renegades and check that grass and leaves change color in different biomes in LOD chunks.

Here's an example, although it might be hard to see without zooming in:
![Terasology-210815201323-1920x1080](https://user-images.githubusercontent.com/13039463/129499686-df3fde89-3528-403f-97fe-0e2eae8ad8aa.png)
